ba-15 Create UI Prototype: Show all inventory items

lyio opened the pull request:
Initial draft for the inventory list with static items

lyio committed:
ba-15 Create UI Prototype: Show all inventory elements
Initial component setup and overview styling

lyio committed:
ba-15 Create UI Prototype: Show all inventory elements
Include socobo-list-item as the list element

lyio committed:
ba-15 Create UI Prototype: Show all inventory elements
Remove search button

lyio commented:
@flashback2k14

flashback2k14 commented on socobo-inventory.html:
```diff
@@ -0,0 +1,109 @@
+
+  <template>
+    <paper-material vertical layout class="inventory" elevation="1">
+      <paper-material elevation="1">
```
please use a toolbar; custom color is possible w/ --paper-toolbar-background and --paper-toolbar-color

flashback2k14 commented on socobo-inventory.html:
```diff
@@ -0,0 +1,109 @@
+      </paper-material>
+    </paper-material>
+
+    <paper-fab style="position:absolute; right: 30px; bottom: 30px" icon="add"></paper-fab>
```
please use css class!

flashback2k14 commented:
comments added in socobo-inventory.html.

lyio committed:
ba-15 Create UI Prototype: Show all inventory elements
Move stlying of FAB to CSS class
Apply css mixins  too paper elements
Use paper-toolbar for the search box

lyio committed:
ba-15 Create UI Prototype: Show all inventory elements
Adjust alignment of search input and checkbox

lyio committed:
ba-15 Create UI Prototype: Show all inventory elements
Adjust padding of the element in regard to external elemets

flashback2k14 commented:
i.O.
